gef-ui/server/controllers/amendments/what-needs-to-change/what-needs-to-change.ts

```diff
@@ -0,0 +1,28 @@
+import { CustomExpressRequest } from '@ukef/dtfs2-common';
+import { Response } from 'express';
+import * as api from '../../../services/api';
+import { WhatNeedsToChangeViewModel } from '../../../types/view-models/amendments/what-needs-to-change-view-model.ts';
+import { asLoggedInUserSession } from '../../../utils/express-session';
+
+export type GetWhatNeedsToChangeRequest = CustomExpressRequest<{
+  params: { dealId: string; facilityId: string; amendmentId: string };
+}>;
+
+export const getWhatNeedsToChange = async (req: GetWhatNeedsToChangeRequest, res: Response) => {
+  try {
+    const { dealId } = req.params;
+    const { userToken } = asLoggedInUserSession(req.session);
+
+    const deal = await api.getApplication({ dealId, userToken });
+
+    const viewModel: WhatNeedsToChangeViewModel = {
+      exporterName: deal.exporter.companyName,
+      previousPage: `/gef/application-details/${dealId}`,
+    };
```

Comment:
we probably do want to pass in the existing values (but that can wait til we've got the GET endpoints up :)

Reply:
Yep, have added a TODO in the code here to make a not of that. Thanks!